WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalUpdateFailedLowBattery.swift

```diff
@@ -49,13 +49,13 @@ private extension CardPresentModalUpdateFailedLowBattery {
         )
 
         static let message = NSLocalizedString(
-            "Updating the reader software failed because the reader’s battery is %.0f%% charged. Please charge the reader above 50%% before trying again.",
+            "Updating the reader software failed because the reader’s battery is %.0f\uFF05 charged. Please charge the reader above 50\uFF05 before trying again.",
             comment: "Button to dismiss the alert presented when an update fails because the reader is low on battery. " +
-                "Please leave the %.0f%% intact, as it represents the current percentage of charge."
+                "Please leave the %.0f\uFF05 intact, as it represents the current percentage of charge."
         )
 
         static let messageNoBatteryLevel = NSLocalizedString(
-            "Updating the reader software failed because the reader is low on battery. Please charge the reader above 50% before trying again.",
+            "Updating the reader software failed because the reader is low on battery. Please charge the reader above 50\uFF05 before trying again.",
```

Comment:
This is the only change that was needed to make the linter happy.

This is because the linter (based on SwiftGen) can't know if the string will go thru `String(format:)` or not before being displayed, so by default it always assumes that it will – which means it expects the string to use `50%%` to escape that `%` and differentiate it from a placeholder. This is why the L10n linter complained here with this false positive. But since this string don't need any parameter to be injected and thus won't go thru `String(format:)`, using an unescaped percent character is what we want, as it won't be parsed or formatted but instead displayed as is unchanged and verbatim.

So using the unicode escape here is the right solution(†) to not get the linter tripped. This would benefit a `// code comment` near this constant though imho.

---

_(†) Another solution would be to use `50%%` to make the linter happy… but then we'd have to use `String(format: messageNoBatteryLevel)` at call site so that this `50%%` gets transformed to `50%` before being displayed… not worth the additional confusing to go that route._

Reply:
For some reason [this one](https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/ar/default/?filters%5Bterm%5D=Updating+the+reader+software+failed+because+the+reader%E2%80%99s+battery+is&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc) keeps failing as well though?

Reply:
Drawback of using `\uFF05` here is that some locales might want to use a different character for percent. Typical example is the arabic locale which has the option to use the dedicated [Arabic Percent Sign](https://www.fileformat.info/info/unicode/char/066a/index.htm) for this. They can totally do that in GlotPress rather than keeping `\uFF05` verbatim in their translation, so that's not a blocker. but it will just be less obvious to the translators that this `\uFF05` is supposed to be a percent character and thus should be replaced by `\u066a` (or directly `٪`) in their translation